robotoff/insights/_enum.py

```diff
@@ -7,21 +7,75 @@ class StrEnum(str, Enum):
 
 @unique
 class InsightType(StrEnum):
+    """InsightType defines the type of the insight."""
+
+    # The 'ingredient spellcheck' insight corrects the spelling in the given ingredients list.
+    # NOTE: this insight is not currently working - last generated insight was in July 2020.
```

Comment:
The import is currently manual for this insight type, which explains why the last generated insight was in July 2020.

Reply:
Thanks, updated the comment!